### ISSUE TITLE
关于模型结构中的问题

### ISSUE THREAD

**Issue by yanni1995** [open-mmlab/mmsegmentation#1329]
请问一下我才用FCN+unet的结构做分割时，我查看网络结构的时候，结构最后同时存在softmax和argmax，请问如何指定保留纯模型数据（不包含argmax和softmax）

**Comment by MengzhangLI** [comment]
Do you mean logits? Check out here:

https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/segmentors/encoder_decoder.py#L239-L241

**Comment by yanni1995** [comment]
> 你是说logits吗？在这里查看：
> 
> https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/segmentors/encoder_decoder.py#L239-L241

assert self.test_cfg.mode in ['slide', 'whole']
        ori_shape = img_meta[0]['ori_shape']
        assert all(_['ori_shape'] == ori_shape for _ in img_meta)
        if self.test_cfg.mode == 'slide':
            seg_logit = self.slide_inference(img, img_meta, rescale)
        else:
            seg_logit = self.whole_inference(img, img_meta, rescale)
        output = F.softmax(seg_logit, dim=1)
修改为：
        output = seg_logit
之后在模型结构中依旧存在softmax。

**Comment by MengzhangLI** [comment]
I mean I just point out key lines rather than babysitter help and you may modify on your own.

**Comment by yanni1995** [comment]
> I mean I just point out key lines rather than babysitter help and you may modify on your own.

我的意思是我改了以后，问题依旧存在，也就是我为什么来ask的目的

**Comment by MengzhangLI** [comment]
Based on your current description, sorry I can not give more information.

Hope you could fix it.

Best,

**Comment by yanni1995** [comment]
> 根据您目前的描述，抱歉，我无法提供更多信息。
> 
> 希望你能修复它。
> 
> 最好的，

意思无法处理是吗，因为我这边修改脚本以后依旧无法删除softmax层，麻烦请工程师帮助一下
